src/twisted/pair/tuntap.py

```diff
@@ -36,8 +37,16 @@
 
 
 _IFNAMSIZ = 16
-_TUNSETIFF = 0x400454CA
-_TUNGETIFF = 0x800454D2
+if (
+    platform.machine() == "parisc"
+    or platform.machine().startswith("ppc")
+    or platform.machine().startswith("sparc")
+):
+    _TUNSETIFF = 0x800454CA  # pragma: nocov
+    _TUNGETIFF = 0x400454D2  # pragma: nocov
```

Comment:
In twisted we had a zero coverage exception policy via coverage.rc

but I think it should be updated.

Also, as documented on coverage.py site the pragma should be `pragma: no coverage`

https://coverage.readthedocs.io/en/6.3.2/excluding.html

```suggestion
):  # pragma: no coverage
    # We don't have CI for parisc, hence no coverage is expected.
    _TUNSETIFF = 0x800454CA
    _TUNGETIFF = 0x400454D2
```

Reply:
Thank you! To be honest, I've copied `pragma: nocov` from `src/twisted/web/test/test_http.py`.

Reply:
Thanks. Those pragma can be updated in another PR if someone is bothered.